### PR TITLE
Zone de commandement général - 1 - Mapping CIC Rework 

### PR DESCRIPTION
**Pull request by LC4492**
# About the pull request

This PR is focused on updating and modifying the Bridge and any command rooms around it (Such as the XO's/CO's Quarters, Synthetic Storage, etc), with the purpose of giving the place a refreshment and, of course, an improvement on efficiency. Feel free to analyse the photos below and to tinker with the map files.

# Explain why it's good for the game

Analyse the map changes and say if its an improvement or not.

# Testing Photographs and Procedure

Captura de Tela (405)

Captura de Tela (406)

Captura de Tela (407)

Captura de Tela (408)

Captura de Tela (409)

image

# Changelog
:cl:
add: Added a little more munitions for the MK1 lockers (2 more normal mags per locker) and some extra boxes of buckshot/slugs to the MK221s lockers.
maptweak: Tweaked the Almayer's Bridge and the rooms surrounding it, such as the PO's, SO's, XO's and CO's quarters and others.
/:cl:

### CODE REVIEW

**Comment by Huffie56** [comment]
if this doesn't get approved you could remove what as been rejected and cut the rest into multiple PR.
i say that because what you propose is a huge change and is spread across the almayer.
been told atomize PR.

File: code/game/jobs/job/logistics/engi/ordnance_tech.dm [hunk 1]
```diff
@@ -33,4 +33,4 @@ AddTimelock(/datum/job/logistics/otech, list(
 /obj/effect/landmark/start/otech
 	name = JOB_ORDNANCE_TECH
 	icon_state = "ot_spawn"
-	job = /datum/job/logistics/otech
+	job = /datum/job/logistics/otech
```
Comment (realforest2001) [inline code/game/jobs/job/logistics/engi/ordnance_tech.dm:36]: Not sure why this file was edited, but it's where the error is

**Review by realforest2001 — CHANGES_REQUESTED** [review]

File: code/game/jobs/job/logistics/engi/ordnance_tech.dm (hunk 1 above)
Comment (LC4492) [reply code/game/jobs/job/logistics/engi/ordnance_tech.dm:36]: For some reason my 'master' version was excluding the otech spawn landmark, so to solve the problem I copied the script lines from the branch in github and glued them on my files, which actually solved the problem, but looks like it have been...too much glued

**Comment by LC4492** [comment]
An 'apollo' console have been added to the CIC and I adjusted some of the required accesses for airlocks and shutters, as they proved to be incoherent in tests

File: code/game/jobs/job/logistics/engi/ordnance_tech.dm (hunk 1 above)
Comment (LC4492) [reply code/game/jobs/job/logistics/engi/ordnance_tech.dm]: I just had forgotten to jump a line in the end, heh

**Comment by LC4492** [comment]
Very well, the changes are prepared, anything holding them from being tested?

**Comment by LilPenpusher** [comment]
As a CO Councillor, I can definitely say that the CO's secure safe (the one currently in the Armoury, which holds the Honour Guard kits) should remain within the CIC and its Armouries, rather than being placed in the CO's office like in the screenshots visible here.

The safe is a valid and quick way for COs to deposit their special-issue combat gear without fear of anyone getting their hands on it (eventhough it's ID-locked, but when has that ever stopped anyone). In the case of a sudden red alert, it means the CO can be out of casual dress and into combat gear within less than a minute and without ever leaving CIC. 

Maybe you can squeeze the safe in? It would be a good thing to keep it close-by.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by c4xmaniac2** [comment]
Como antiguo consejo de CO, puedo confirmar que estos cambios de CIC no son buenos. Por favor, reconsidere cambiar las cosas que no necesitan cambiarse. El CIC actual lo tiene todo bien diseñado, este diseño complica las cosas innecesariamente.

**Comment by Huffie56** [comment]
you should ask for help to fix you merge conflict on discord.

**Comment by LC4492** [comment]
> Como antiguo consejo de CO, puedo confirmar que estos cambios de CIC no son buenos. Por favor, reconsidere cambiar las cosas que no necesitan cambiarse. El CIC actual lo tiene todo bien diseñado, este diseño complica las cosas innecesariamente.

Legal

**Comment by LC4492** [comment]
> you should ask for help to fix you merge conflict on discord.

There's no point on fixing any conflicts if the research's PR is still stuck on testmerge, when it gets approved I will need to update the map anyway, so doing so now will only make me do it again in the future.

**Comment by LC4492** [comment]
> Como antiguo consejo de CO, puedo confirmar que estos cambios de CIC no son buenos. Por favor, reconsidere cambiar las cosas que no necesitan cambiarse. El CIC actual lo tiene todo bien diseñado, este diseño complica las cosas innecesariamente.

Aliás, você certamente poderia especificar onde exatamente você vê o problema e o que está sendo complicado, preferencialmente em inglês

**Comment by Nanu308** [comment]
> > you should ask for help to fix you merge conflict on discord.
> 
> There's no point on fixing any conflicts if the research's PR is still stuck on testmerge, when it gets approved I will need to update the map anyway, so doing so now will only make me do it again in the future.

Research PR is merged, go ahead and deconflict for TM

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by LC4492** [comment]
> > > you should ask for help to fix you merge conflict on discord.
> > 
> > 
> > There's no point on fixing any conflicts if the research's PR is still stuck on testmerge, when it gets approved I will need to update the map anyway, so doing so now will only make me do it again in the future.
> 
> Research PR is merged, go ahead and deconflict for TM

Everything ready to run

**Comment by morrowwolf** [comment]
image

**Review by realforest2001 — CHANGES_REQUESTED** [review]
See above. Also, CIC armory button is linked to the other armouries, not CIC.

**Comment by realforest2001** [comment]
image
Can get locked in here.

image
This set of doors, and the ones on the other side of CIC, have double layers of CIC blast doors.

image
image
Splitting the armouries so CIC can open is useful, but it also leaves no indication that the others are open or closed. I would suggest moving the CO's safe to replace one of these SO lockers, and have the North/South facing doors use CIC armoury, and the rest of the CIC armoury shutters use the main ship one.

**Comment by realforest2001** [comment]
image
Undocumented change to armoury contents, same goes for the shotgun side.

**Comment by realforest2001** [comment]
image
Need to document your changes more than a single line changelog. This extra layer of blast doors all over the place controlled from this bubble seems unnecessary.

**Review by morrowwolf — CHANGES_REQUESTED** [review]
Random assortment of things that need changed:
HG kit needs to be back in CIC, not the CO's personal locker in his room. (I do not care if a CO councilor said otherwise)

Ammo changes need logged (although I do not mind the change to 1:2 AP:normal split). The shotgun lockers do need more buckshot/slug containers. Probably 2 of each per locker.

SOs *must* be within sight of each other. The center bubble will likely need to be made smaller.

The ladders west side of CIC are used by all sorts of people and having to walk through an already crowded and tight space to get to them now is not good. People should not be encouraged to walk through CIC. This must change.

CO hypersleep should be with the general command crew, not in their room.

Jones need his bed back.

My first pass opinion is the separated overwatch consoles will make non-4 SO structures far harder and needlessly annoying (especially considering that 4 SOs is rare outside of peak hours). You must justify this change to be better than the previous design or change it so two SOs can control four consoles.

Otherwise:

Officer study is neat.
New XO room looks good.
Synth closet is tight but functional.
Command power substation looks good.
CO office looks good (other than Jones bed)

**Comment by LC4492** [comment]
> image Need to document your changes more than a single line changelog. This extra layer of blast doors all over the place controlled from this bubble seems unnecessary.
> 
> image MP lockdown on here is controlled by a button on the complete other side of CIC, which makes it relatively redundant.

The Military Police lockdown is meant to be used, mainly, in shipside situations where the CIC needs to be protected, such as in a boarding-party attack. It is meant to be a delaying mechanism, so the Military Police can, as an example, lockdown the back and/or side doors and be able to keep their eyes straight to the front, without the commander needing to completely lockdown CIC and block any type of crew traffic that could ever appear. The frontal lockdown is meant for the MP or any other command staff on the watch in the post to be able to delay the invading forces while they warn the commanding team on duty about the current threat.

The rest of the observations are correct and I will see the possibilities

**Comment by LC4492** [comment]
> Random assortment of things that need changed: HG kit needs to be back in CIC, not the CO's personal locker in his room. (I do not care if a CO councilor said otherwise)
> 
> Ammo changes need logged (although I do not mind the change to 1:2 AP:normal split). The shotgun lockers do need more buckshot/slug containers. Probably 2 of each per locker.
> 
> SOs _must_ be within sight of each other. The center bubble will likely need to be made smaller.
> 
> The ladders west side of CIC are used by all sorts of people and having to walk through an already crowded and tight space to get to them now is not good. People should not be encouraged to walk through CIC. This must change.
> 
> CO hypersleep should be with the general command crew, not in their room.
> 
> Jones need his bed back.
> 
> My first pass opinion is the separated overwatch consoles will make non-4 SO structures far harder and needlessly annoying (especially considering that 4 SOs is rare outside of peak hours). You must justify this change to be better than the previous design or change it so two SOs can control four consoles.
> 
> Otherwise:
> 
> Officer study is neat. New XO room looks good. Synth closet is tight but functional. Command power substation looks good. CO office looks good (other than Jones bed)


> Random assortment of things that need changed: HG kit needs to be back in CIC, not the CO's personal locker in his room. (I do not care if a CO councilor said otherwise)
> 
> Ammo changes need logged (although I do not mind the change to 1:2 AP:normal split). The shotgun lockers do need more buckshot/slug containers. Probably 2 of each per locker.
> 
> SOs _must_ be within sight of each other. The center bubble will likely need to be made smaller.
> 
> The ladders west side of CIC are used by all sorts of people and having to walk through an already crowded and tight space to get to them now is not good. People should not be encouraged to walk through CIC. This must change.
> 
> CO hypersleep should be with the general command crew, not in their room.
> 
> Jones need his bed back.
> 
> My first pass opinion is the separated overwatch consoles will make non-4 SO structures far harder and needlessly annoying (especially considering that 4 SOs is rare outside of peak hours). You must justify this change to be better than the previous design or change it so two SOs can control four consoles.
> 
> Otherwise:
> 
> Officer study is neat. New XO room looks good. Synth closet is tight but functional. Command power substation looks good. CO office looks good (other than Jones bed)

Hm...I didn't saw any beds belonging to Jones when I started to work on the rework, I don't know what is that

**Comment by LC4492** [comment]
image

This way all four SOs can see eachother with ease, now.

**Comment by lcarapaica** [comment]
Untitled
-There is an undocumented blast shutter in the synthetics closet, seems unnecessary.
-I suggest the XO's bedroom door face towards the inner CIC (western wall) rather than the unsecure hallway, considering the CO's and SO's does the same.

**Comment by LC4492** [comment]
> Untitled -There is an undocumented blast shutter in the synthetics closet, seems unnecessary. -I suggest the XO's bedroom door face towards the inner CIC (western wall) rather than the unsecure hallway, considering the CO's and SO's does the same.

It seems unnecessary because is missplaced.

**Comment by nauticall** [comment]
Sorry, but I'm not a fan, to be honest. The current iteration of the CIC is deliberately designed to give officers access to all the essential computers and tools within two to five tile's worth of walking from each other. With the current CIC, you can man multiple overwatch consoles at the same time, and the CO/XO only has to walk a short bit to make an announcement towards the computer.

This layout _looks_ pretty, but practically speaking it drastically increases travel time of officers to get from A. between overwatch consoles and B. to the operations (announcement) computers. Command's job is **communication** and **intelligence**, and extra seconds spent commuting from computer to computer can spell the difference between victory and defeat as they experience delays in gathering information. Not to mention lowpop too, where having a single XO running CIC is a regular occurrence. They can manage just fine with the existing CIC, but will have to run a marathon with this layout. Charlie asks for a critical SL change while Delta SL requests an OB -- with the current layout an officer can do both simultaneously, but with this the XO will have to run back and forth to get things done.

Overall, I get the effort you've put into making this and it looks nice. But it just isn't practical for gameplay purposes and myself as well as a few other community members have stated their dislike of the layout. The CIC is fine as it it is right now.

**Comment by LC4492** [comment]
> Sorry, but I'm not a fan, to be honest. The current iteration of the CIC is deliberately designed to give officers access to all the essential computers and tools within two to five tile's worth of walking from each other. With the current CIC, you can man multiple overwatch consoles at the same time, and the CO/XO only has to walk a short bit to make an announcement towards the computer.
> 
> This layout _looks_ pretty, but practically speaking it drastically increases travel time of officers to get from A. between overwatch consoles and B. to the operations (announcement) computers. Command's job is **communication** and **intelligence**, and extra seconds spent commuting from computer to computer can spell the difference between victory and defeat as they experience delays in gathering information. Not to mention lowpop too, where having a single XO running CIC is a regular occurrence. They can manage just fine with the existing CIC, but will have to run a marathon with this layout. Charlie asks for a critical SL change while Delta SL requests an OB -- with the current layout an officer can do both simultaneously, but with this the XO will have to run back and forth to get things done.
> 
> Overall, I get the effort you've put into making this and it looks nice. But it just isn't practical for gameplay purposes and myself as well as a few other community members have stated their dislike of the layout. The CIC is fine as it it is right now.

Making it a nightmare segment would make it viable? It could be set to only be used on higher pops, such as 150 to 160 players

**Comment by LC4492** [comment]
I may be able to easily make two versions, one which is fitted with a scheme in a smaller fashion, for small pop rounds, and this one, which is bigger and can certainly work better on bigger pops.

**Comment by Huffie56** [comment]
i am suggesting you separate your change into multiple PR's and use this PR as resource to use to make them.
the first one to make could be the change you made on the older officer office south of the pods...
image
nobody use those...
the second one could be CO quarter but if i was you i would ask to the CO whitelist players on what could be improve...

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Pull request closed without merging**